iOS/Info.plist

```diff
@@ -69,5 +69,10 @@
 		<string>UIInterfaceOrientationLandscapeLeft</string>
 		<string>UIInterfaceOrientationLandscapeRight</string>
 	</array>
+    <key>NSAppTransportSecurity</key>
+    <dict>
+        <key>NSAllowsArbitraryLoads</key>
```

Comment:
If the link is embedded in the KML and we don't own the KML file then we do need this.

Reply:
The alternative is downloading the `https` versions of the files from [NOAA](https://www.wpc.ncep.noaa.gov/kml/noaa_chart/) ([legend image](https://www.wpc.ncep.noaa.gov/kml/noaa_chart/legend_GE_d1.png), [no sigwx image](https://www.wpc.ncep.noaa.gov/kml/noaa_chart/no_sigwx_d1.png)) manually and overlaying them on top of the map. That might get not work super well with the "No significant Weather Is Expected" image, though, as it won't move with the map as it does when being part of the layer.

Reply:
I think I would leave it. But get an opinion from @dg0yal

Reply:
[We didn't have to do this for the old sample viewer.](https://github.com/Esri/arcgis-runtime-samples-ios/blob/b2694f9e5a97cd81ea042c1078708215ab4f3f48/arcgis-ios-sdk-samples/Info.plist#L52-L71) We should figure out which domains we need exceptions for and only allow those.

Reply:
I added an exception for `noaa.gov` like in the old sample viewer, as Phil suggested.